File: python/bc1072.py
iVec = int(input())
cIn = 0
cOut = 0

for i in range(0, iVec):
    val = int(input())

    if val >= 10 and val <= 20: cIn += 1
    else: cOut += 1

print(f"{cIn} in\n{cOut} out")

File: cpp/bc1072.cpp
#include <iostream>

using namespace std;

int main()
{
	int iVec, cIn, cOut, val;
	cIn = cOut = 0;
	
	cin >> iVec;
	
	for(int i = 0; i < iVec; i++)
	{
		cin >> val;
		val >= 10 && val <= 20? cIn++ : cOut++;
	}
	
	cout << cIn << " in\n" << cOut << " out" << endl;
}
